navigation/src/test/java/com/graphhopper/navigation/NavigateResponseConverterTest.java

```diff
@@ -273,7 +273,7 @@ public void intersectionTest() {
 
         step = steps.get(4);
         intersection = step.get("intersections").get(3);
-        assertEquals(1, intersection.get("in").asInt());
+        assertEquals(2, intersection.get("in").asInt());
         assertEquals(0, intersection.get("out").asInt());
```

Comment:
This is the intersection:

![image](https://user-images.githubusercontent.com/17603532/205175167-69ead8a9-1696-4d34-bd87-adf7533f754b.png)

Why did we expect zero outgoing roads so far? It looks like one can leave this intersection in Southern direction. The reason there used to be one incoming road and now there are two is that the road coming in from the West is tagged as `highway=pedestrian` and thus was excluded for the car profile so far. But since this is a oneway shouldn't this rather be an outgoing road?

Reply:
Ok so the indices simply changed because now the pedestrian route is accepted and nothing else needs to be done here? Basically this test only makes sure nothing changes, because we can't tell which indices are 'correct'?

Reply:
Yes, probably.